packages/js/src/helpers/get-premium-benefits.js

```diff
@@ -0,0 +1,13 @@
+import { __ } from "@wordpress/i18n";
+
+/**
+ * @returns {string[]} The premium benefits.
+ */
+export const getPremiumBenefits = () => [
+	`<strong>${ __( "AI", "wordpress-seo" ) }:</strong> ${ __( "Better SEO titles and meta descriptions, faster", "wordpress-seo" ) }`,
+	`<strong>${ __( "Multiple keywords", "wordpress-seo" ) }:</strong> ${ __( "Rank higher for more searches", "wordpress-seo" ) }`,
+	`<strong>${ __( "Super fast", "wordpress-seo" ) }</strong> ${ __( "internal linking suggestions", "wordpress-seo" ) }`,
```

Comment:
I'm afraid a line like this would be a mess for translators. For example in Italian it would be the other way around:
`suggerimenti di link interni <strong>super veloci</strong>`
We should probably use placeholders for the `<strong>` tags.